ibm/service/power/resource_ibm_pi_volume.go

```diff
@@ -240,6 +254,17 @@ func resourceIBMPIVolumeCreate(ctx context.Context, d *schema.ResourceData, meta
 		replicationEnabled := v.(bool)
 		body.ReplicationEnabled = &replicationEnabled
 	}
+	if v, ok := d.GetOk(Arg_ReplicationSites); ok {
+		if d.Get(Arg_ReplicationEnabled).(bool) {
+			replicationSites := make([]string, len(v.([]interface{})))
+			for i, site := range v.([]interface{}) {
+				replicationSites[i] = site.(string)
+			}
+			body.ReplicationSites = replicationSites
```

Comment:
`MinItems:    1,` is only enforce if the argument is required otherwise it default to `0` for optional ones.
Could you add check for `len > 0` before setting it?
And I this can be simply with something like this
```
if len(v.([]interface{})) > 0 {
			body.ReplicationSites = flex.ExpandStringList(v.([]interface{}))
		}

```